client/components/catalog/Card.vue

```diff
@@ -34,16 +43,18 @@
 </template>
 
 <script>
+import { mapActions, mapGetters } from 'vuex';
 import first from 'lodash/first';
 import get from 'lodash/get';
 import { getSchema } from 'shared/activities';
-import { mapActions } from 'vuex';
 
 export default {
   props: {
     repository: { type: Object, required: true }
   },
   computed: {
+    ...mapGetters(['isAdmin']),
+    ...mapGetters('course', ['isCourseAdmin']),
```

Comment:
I think this won't work since these getters are inited after you go to the specific course context.